Revised UI tweaks

persi-persu opened the pull request.

Commit:
Add an all movie row & row visibility settings (1)

Commit:
Add row visibility settings (2)

Commit:
Add row visibility settings (3)

Commit:
Add row visibility settings (4)

Poor wording

courville committed:
Merge pull request #5 from persi-persu/nova

 Add an all movie row & row visibility settings

Commit:
Fix typo

courville committed:
Merge pull request #6 from persi-persu/nova

Fix typo

courville committed:
Minor rewording

courville committed:
Even better.

courville committed:
Crowdin sync to get fr translations corresponding to latest leanback changes.

courville committed:
Changelog++

courville committed:
Compact changelog

Commit:
Add play button to tv show (1)

Commit:
Add play button to tv show (2)

courville committed:
Merge pull request #7 from persi-persu/nova

Add play button to tv show

courville committed:
Merge branch 'nova' of https://github.com/nova-video-player/aos-Video into nova

PhieF committed:
sometimes cloud file are not indexed : needs path

Commit:
Fix all movie row

courville committed:
Merge pull request #10 from persi-persu/patch-2

Fix all movie row

Commit:
Make text bigger in tv player

Commit:
Add "Genres: " before genres

Commit:
Change select level of header

Commit:
Change select level of header

Commit:
Change select level of header

- Add left padding to header

Commit:
Change select level of header

- Add "Genres: " before genres

Commit:
Change select level of header

Commit:
Show directory path without "/" at end

Commit:
Use same title as category

Commit:
Add year and rating to tv shows more details

Commit:
Use same title as category

Commit:
Use same title as category

Commit:
Show directory path instead of file path

Commit:
Change padding of get subtitles button

Commit:
Add year and rating to tv shows more details

Commit:
Add tv shows infos to recommendations

Commit:
Change channels position

Commit:
Change tv shows infos format in player

Commit:
Change select level of header

- Show directory path instead of file path
- Fix bottom padding

courville commented:
Thanks for the PR and the revised commits with more detailed on what it is intended to.
I think these are great additions on the leanback (Android TV) UI.
The only issue I have for now, is that I would like to preserve a parity in the UI look at feel between the phone/tablet and TV.
Would you feel ok to propagate the changes also in the non-leanback part of the code?

Commit:
Add deletion of seasons

Commit:
Add confirm delation

- Minor text change to description
- Add deindexation of seasons
- Add deletion of seasons

Commit:
Disable network bookmarks by default

Commit:
Add deindexation of seasons 

- Add deletion of seasons

Commit:
Add confirm deletion

Commit:
Add confirm deletion

Commit:
Add deindexation of seasons

- Add deletion of seasons

Commit:
Add deindexation of seasons

- Add deletion of seasons

Commit:
Add deindexation of seasons

- Add deletion of seasons

Commit:
Add deindexation of seasons

- Add deletion of seasons

Commit:
Fix displaying unknown year

Commit:
Add deindexation of seasons

- Add deletion of seasons
- Fix deletion of season rows

Commit:
Fix displaying unknown year

Commit:
Add deindexation of seasons

- Add deletion of seasons

Commit:
Add tv menu hide animation

Commit:
Add tv menu hide animation

Commit:
Add deindexation of seasons

- Add deletion of seasons

Commit:
Fix typo

Commit:
Fix typo

Commit:
Fix typo

Commit:
Fix typo

ghost commented:
I do not really use the non-leanback UI. I looked a bit but it is very different.
Here are some new changes.

Commit:
- Change tv player layout

Commit:
- Change tv player layout

Commit:
- Change tv player layout

Commit:
Add "-" before remaining time

Commit:
Change to player slider

Commit:
Change to player slider

Commit:
Change to tv player layout

Commit:
Change to tv player layout

Commit:
Change to tv player layout

Commit:
Fix cast row layout

Commit:
Fix duration info

Commit:
Fix cast row layout

Commit:
Fix subtitles row layout

Commit:
Fix cast row layout

Commit:
Fix plot row layout

Commit:
Move decoder info to tv menu

Commit:
Change switchable menu item layout

Commit:
Move decoder info to tv menu

Commit:
Merge audio menu items

- Move decoder info to tv menu

Commit:
Fix tv menu focus

Commit:
Move decoder info to tv menu

Commit:
Fix tv menu focus

Commit:
Remove redundant info

courville commented on src/main/java/com/archos/mediacenter/video/player/PlayerController.java:
```diff
@@ -1265,7 +1265,7 @@ private int setProgress() {
                     mProgress2.setProgress((int) pos);
                 currentText = stringForTime(position);
-                endText = stringForTime(duration-position > 0 ? duration-position : 0);
+                endText = "-" + stringForTime(duration-position > 0 ? duration-position : 0);
```
I prefer preserving the current view since it is explicit enough to have the remaining time on the right and there is no such thing that negative time...

Commit:
Add audio badges

Commit:
Minor text change

- Add all episodes button

Commit:
Add audio badges

Commit:
Add all episodes button

Commit:
Add all episodes button

Commit:
Add PIP playback

courville commented on src/leanback/java/com/archos/mediacenter/video/leanback/presenter/ShortcutDetailsPresenter.java:
```diff
@@ -24,6 +24,10 @@
-        viewHolder.getSubtitle().setText(shortcut.getFriendlyUri());
+        
+        String uri = shortcut.getFriendlyUri();
+        String newUri = uri.substring(0, uri.length() - 1);
```
perhaps better to remove specifically "/" instead of the systematically the last char

courville replied:
Need to check if all the shortcuts smb/upnp/sftp etc.. has an ending /

courville commented on res/values/strings.xml:
```diff
@@ -135,7 +136,7 @@
     <string name="scrap_remove">Remove the description</string>
     <string name="scrap_remove_short">Remove</string> <!-- for small screens -->
-    <string name="scrap_change">Change info for all episodes</string>
+    <string name="scrap_change">Change the description for all episodes</string>
```
we prefer info

courville commented on res/xml/preferences_video.xml:
```diff
@@ -210,7 +210,7 @@
 
         <CheckBoxPreference
-            android:defaultValue="true"
+            android:defaultValue="false"
```
I prefer to keep is true by default

courville commented:
Just saw your discussions on irc channel, please do not force push any change on the current PR.
I am git cherry-picking into nova all the changes that we find ok (we are doing code review with Phie).
Once merged let's treat the other as new individual PR.
Drop in on irc fore more if need be.

Commit:
Darken backdrop

Commit:
Change smb credentials view

- Change licences view

Commit:
Change "description" to "info"

- Minor text changes
- Add movie trailers setting

Commit:
Update licence url

Commit:
Add movie trailers setting

- Minor text change

Commit:
Add movie trailers setting

Commit:
Fix all movies row

Commit:
Reduce backdrop darkening

Commit:
Fix showed directory path

Commit:
Change smb credentials view

Commit:
Add movie trailers setting

- Fix leanback UI category

Commit:
Change licences view

Commit:
Minor text change

Commit:
Minor text change

Commit:
Minor text change

- Add all movie row sort setting

Commit:
Add all movie row sort setting

Commit:
Add all movie row sort setting

Commit:
Add all movie row sort setting

Commit:
Add all movie row sort setting

Commit:
Change all tv show row sort setting

- Change tv show sort

Commit:
Change tv show sort

Commit:
Change tv show sort

Commit:
Change all tv show row sort setting

Commit:
Change all tv show row sort setting

- Add all movie row sort setting

courville commented on res/layout/card_layout.xml:
```diff
@@ -52,6 +52,20 @@
                         android:textSize="@dimen/tv_menu_font_size" />
                 </LinearLayout>
             </RelativeLayout>
+            
```
I think this is a nice idea to populate the info card with more valuable information but I would add more info on codecs first then hwdec or swdec without details.

courville commented on res/xml/preferences_video.xml:
```diff
@@ -93,7 +93,7 @@
 
     </PreferenceCategory>
-    <PreferenceCategory android:title="@string/preferences_user_interface" android:key="category_leanback_user_interface" >
+    <PreferenceCategory android:title="@string/preferences_leanback_user_interface" android:key="category_leanback_user_interface" >
```
I will not merge this change since the setting applies both to AndroidTV and phones/tablets. This is thus not intended to TV only.

Commit:
Add list deletion

Commit:
Add list deletion

Commit:
Reduce decoder info

Commit:
Reduce decoder info

Commit:
Reduce decoder info

Commit:
Change role format

Commit:
Change role format

Commit:
Change role format

Commit:
Fix change role format

Commit:
Fix all movie row sort setting

- Fix all tv show row sort setting

courville commented:
I found a way to bundle your commits using git rebase -i and the squash functionality.
It will look better on git history.
However it requires me to push and rewrite github git history.
This means that you will have to do a full repo init afterwards to get in sync otherwise there will be issues.
If this is ok, I will do it tonight.

courville commented:
@persi-persu @PhieF @phhusson @av500 @nnoury I forced pushed my current Video git repo to github where I squashed may commits to look good. This will interfere with your current already pulled versions.
Be warned ;-)
